meetup/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from pizzaplace.models import PizzaPlace
+from django.core.validators import RegexValidator
+from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from django.core.exceptions import ValidationError
+
+
+def validate_urlname(link):
+  validator = RegexValidator(
+    regex='meetup\.com\/\w+(-\w+)*\/$',
+    message="Does not conform to Meetup Url",
+    code='invalid_url')
+  return validator(link)
+
+def validate_meetup_exists(link):
+  looker = MeetupApiLookupAgent(link)
+  is_real = looker.is_real_meetup()
+
+  if not is_real:
+    raise ValidationError("That's not a meetup")
+
```

Comment:
Standard style convention in Python is two lines between classes. There's actually an official style guide for this stuff: https://www.python.org/dev/peps/pep-0008/. Let's add a story soon to add a code linter, which catches nitpicks like this.